### ISSUE TITLE
When the disk where the /tmp directory is located is full, a large number of empty exec process * * * files are generated and continue to accumulate and grow

### ISSUE THREAD

**Issue by lidajia** [cri-o/cri-o#7604]
### What happened?

When the disk where the/tmp directory is located is full, a large number of empty exec process * * * files are generated and continue to accumulate and grow.

### What did you expect to happen?

When the disk where the /tmp directory is located is full, the exec operation should only fail and there should be no leakage of the exec-process** file

### How can we reproduce it (as minimally and precisely as possible)?

Use large files to fill the disk to which the/tmp directory belongs，then running pod with exec type livenesscheck.

### CRI-O and Kubernetes version

<details>

```console
$ crio --version
# paste output here
```

```console
$ kubectl version --output=json
# paste output here
```
1.27.0
</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here
```
centos 7.9
</details>

### Additional environment details (AWS, VirtualBox, physical, etc.)

<details>
centos 7.9
</details>

**Comment by saschagrunert** [comment]
Hey @lidajia, thank you for the issue report! I assume this is exec probe related? The files are being created there: 

https://github.com/cri-o/cri-o/blob/2e1ba2771e042ce9d24d3e0b053fb0491ba695bb/internal/oci/runtime_oci.go#L1263-L1266

Usually they should be cleaned-up in https://github.com/cri-o/cri-o/blob/2e1ba2771e042ce9d24d3e0b053fb0491ba695bb/internal/oci/runtime_oci.go#L403-L407

Can you share some error logs or similar so that I can follow the path?

**Comment by lidajia** [comment]
fix in #7603
